modules/covid_vaccine/spec/jobs/covid_vaccine/registration_email_job_spec.rb

```diff
@@ -20,7 +20,7 @@
     end
 
     it 'executes perform' do
-      instance = instance_double(VaNotify::Service, send_email: nil)
+      instance = instance_double(VaNotify::Service, send_email: { id: '123456789' })
```

Comment:
should this id be a variable or a `let`?

Reply:
it should probably be more of a: `expect_any_instance_of(VaNotify::Service).to_receive(:send_mail).with(...).and_return({ id: '123456789' }).once`